cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -605,43 +615,35 @@ define(["jquery", "edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers", "common/j
             });
 
             describe("Subsection", function() {
-                var getDisplayNameWrapper, setEditModalValues, mockServerValuesJson,
-                    selectDisableSpecialExams, selectBasicSettings, selectAdvancedSettings,
-                    selectAccessSettings, selectTimedExam, selectProctoredExam, selectPracticeExam,
+                var getDisplayNameWrapper, setEditModalValues, setContentVisibility,  mockServerValuesJson,
+                    selectDisableSpecialExams, selectTimedExam, selectProctoredExam, selectPracticeExam,
                     selectPrerequisite, selectLastPrerequisiteSubsection, checkOptionFieldVisibility;
 
                 getDisplayNameWrapper = function() {
                     return getItemHeaders('subsection').find('.wrapper-xblock-field');
                 };
 
-                setEditModalValues = function (start_date, due_date, grading_type, is_locked) {
+                setEditModalValues = function (start_date, due_date, grading_type) {
                     $("#start_date").val(start_date);
                     $("#due_date").val(due_date);
                     $("#grading_type").val(grading_type);
-                    $("#staff_lock").prop('checked', is_locked);
-                };
-
-                selectDisableSpecialExams = function() {
-                    this.$("input.no_special_exam").prop('checked', true).trigger('change');
-                };
-
-                selectBasicSettings = function() {
-                   this.$(".modal-section .settings-tab-button[data-tab='basic']").click();
                 };
 
-                selectAdvancedSettings = function() {
-                   this.$(".modal-section .settings-tab-button[data-tab='advanced']").click();
+                setContentVisibility = function (visibility) {
+                    $('input[name=content-visibility][value='+visibility+']').prop('checked', true);
                 };
 
-                selectAccessSettings = function() {
-                   this.$(".modal-section .settings-tab-button[data-tab='access']").click();
+                selectDisableSpecialExams = function() {
+                    this.$("input.no_special_exam").prop('checked', true).trigger('change');
                 };
 
                 selectTimedExam = function(time_limit, hide_after_due) {
                     this.$("input.timed_exam").prop('checked', true).trigger('change');
                     this.$(".field-time-limit input").val(time_limit);
                     this.$(".field-time-limit input").trigger('focusout');
-                    this.$('.field-hide-after-due input').prop('checked', hide_after_due).trigger('change');
+                    if (hide_after_due) {
```

Comment:
I don't think so, but correct me if this line of thinking is wrong.

- Some timed exams already have `hide_after_due` set to True, but the default is false.
- With the changes being made in this PR, we will stop setting this field on timed exams (it won't be added to their database model). I need to double check to confirm this.
- `hide_after_due` is being set at the subsection level now

Given that, my thinking is that when a timed exam has `hide_after_due` set to True, it can override the xblock default visibility value. If it's not set, we should let the changes I'm making in this PR take precedence - having the default value of False is not an indicator that a timed exam is explicitly choosing to do anything with visibility.